woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -934,3 +936,302 @@ def test_dataframe_methods_on_accessor_other_returns(sample_df):
+
+    df_tag_instantiated = schema_df.ww.select('Datetime')
+    assert len(df_tag_instantiated.columns) == 1
+    assert 'signup_date' in df_tag_instantiated.columns
+
+
+def test_select_list_inputs(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    schema_df = sample_df.copy()
+    schema_df.ww.init(time_index='signup_date',
+                      index='id',
+                      name='df_name',
+                      logical_types={
+                          'full_name': FullName,
+                          'email': EmailAddress,
+                          'phone_number': PhoneNumber,
+                          'signup_date': Datetime(datetime_format='%Y-%m-%d'),
+                      },
+                      semantic_tags={
+                          'full_name': ['new_tag', 'tag2'],
+                          'age': 'numeric',
+                          'signup_date': 'date_of_birth',
+                          'email': 'tag2',
+                          'is_registered': 'category'
+                      })
+
+    df_just_strings = schema_df.ww.select(['FullName', 'index', 'tag2', 'boolean'])
+    assert len(df_just_strings.columns) == 4
+    assert 'id' in df_just_strings.columns
+    assert 'full_name' in df_just_strings.columns
+    assert 'email' in df_just_strings.columns
+    assert 'is_registered' in df_just_strings.columns
+
+    df_mixed_selectors = schema_df.ww.select([FullName, 'index', 'time_index', Integer])
+    assert len(df_mixed_selectors.columns) == 4
+    assert 'id' in df_mixed_selectors.columns
+    assert 'full_name' in df_mixed_selectors.columns
+    assert 'signup_date' in df_mixed_selectors.columns
+    assert 'age' in df_mixed_selectors.columns
+
+    df_common_tags = schema_df.ww.select(['category', 'numeric', Boolean, Datetime])
+    assert len(df_common_tags.columns) == 3
+    assert 'is_registered' in df_common_tags.columns
+    assert 'age' in df_common_tags.columns
+    assert 'signup_date' in df_common_tags.columns
+
+
+def test_select_semantic_tags_no_match(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    schema_df = sample_df.copy()
+    schema_df.ww.init(time_index='signup_date',
+                      index='id',
+                      name='df_name',
+                      logical_types={
+                          'full_name': FullName,
+                          'email': EmailAddress,
+                          'phone_number': PhoneNumber,
+                          'signup_date': Datetime(datetime_format='%Y-%m-%d'),
+                      }, semantic_tags={
+                          'full_name': ['new_tag', 'tag2'],
+                          'age': 'numeric',
+                          'signup_date': 'date_of_birth',
+                          'email': 'tag2'
+                      })
+
+    assert len(schema_df.ww.select(['doesnt_exist']).columns) == 0
+
+    df_multiple_unused = schema_df.ww.select(['doesnt_exist', 'boolean', 'category', PhoneNumber])
```

Comment:
One test I didn't see is to make sure duplicate items in `select` doesn't break Schema.
```
schema_df.ww.select(['full_name', FullName, 'FullName'])
```

Reply:
makes sense--will add